backend/api/pretix/tests/test_update_attendee_ticket.py

```diff
@@ -0,0 +1,228 @@
+from django.test import override_settings
+
+
+@override_settings(PRETIX_API="https://pretix/api/")
+def test_cannot_update_ticket_if_i_am_not_the_owner(
+    graphql_client, requests_mock, user, conference_factory, pretix_user_tickets
+):
+    graphql_client.force_login(user)
+    conference = conference_factory(pretix_organizer_id="org", pretix_event_id="event")
+    pretix_user_tickets[0]["attendee_email"] = "not@my.com"
+
+    requests_mock.get(
+        f"https://pretix/api/organizers/org/events/event/tickets/attendee-tickets?attendee_email={user.email}",
+        json=pretix_user_tickets,
+    )
+
+    query = """
+    mutation UpdateTicket($conferenceCode: String!, $input: UpdateAttendeeTicketInput!) {
+        updateAttendeeTicket(conferenceCode: $conferenceCode, input: $input) {
+            ... on AttendeeTicket {
+                id
+            }
+        }
+    }
+    """
+
+    response = graphql_client.query(
+        query,
+        variables={
+            "conferenceCode": conference.code,
+            "input": {
+                "id": pretix_user_tickets[0]["id"],
+                "name": "Ester",
+                "email": "ester@pycon.it",
+            },
+        },
+    )
+
+    assert (
+        response["errors"][0]["message"] == "You are not allowed to update this ticket."
+    )
+
+
+@override_settings(PRETIX_API="https://pretix/api/")
+def test_update_ticket_errors_handling(
+    graphql_client, mocker, requests_mock, user, conference_factory
+):
+    graphql_client.force_login(user)
+    conference = conference_factory(pretix_organizer_id="org", pretix_event_id="event")
+    mocker.patch("pretix.is_ticket_owner", return_value=True)
+
+    requests_mock.patch(
+        "https://pretix/api/organizers/org/events/event/orderpositions/999/",
+        text="Not Found",
+        status_code=404,
+    )
+
+    query = """
+    mutation UpdateTicket($conferenceCode: String!, $input: UpdateAttendeeTicketInput!) {
+        updateAttendeeTicket(conferenceCode: $conferenceCode, input: $input) {
+            ... on AttendeeTicket {
+                id
+            }
+        }
+    }
+    """
+
+    response = graphql_client.query(
+        query,
+        variables={
+            "conferenceCode": conference.code,
+            "input": {
+                "id": "999",
+                "name": "Ester",
+                "email": "ester@pycon.it",
+            },
+        },
+    )
+
+    assert "404 Client Error" in response["errors"][0]["message"]
+
+
+@override_settings(PRETIX_API="https://pretix/api/")
+def test_update_ticket(
```

Comment:
What is the original name that you change in this test?

Reply:
PRETIX_API: https://tickets.pycon.it/api/v1/